src/components/Project/ProjectNew.js

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {withRouter} from 'react-router';
+import Select from 'react-select';
+
+const styles = {
```

Comment:
Can we store css in a separate file?

Reply:
This is a bit different to a normal CSS file as it has a higher hierarchy so will override's UI styles. I really like all the code being together in while file, I feel it's super clean